doc/meshes_models.asc

```diff
@@ -123,16 +123,20 @@ and the latter is used for collision testing.
 ----
 struct tr_staticmesh   // 32 bytes
 {
-    uint32_t   ID;   // Static Mesh Identifier
-    uint16_t   Mesh; // Mesh (offset into MeshPointers[])
-    tr2_vertex VisibilityBox[2];
-    tr2_vertex CollisionBox[2];
-    uint16_t   Flags;
+    uint32_t        ID;   // Static Mesh Identifier
+    uint16_t        Mesh; // Mesh (offset into MeshPointers[])
+    tr_bounding_box VisibilityBox;
+    tr_bounding_box CollisionBox;
+    uint16_t        Flags;
+};
+
+struct tr_bounding_box // 12 bytes
+{
+    int16_t MinX, MaxX, MinY, MaxY, MinZ, MaxZ; 
 };
 ----
 
-For `VisibilityBox` and `CollisionBox`, dimensions are specified using two vectors, first being the `bounding box minimum`, and second being the `bounding box
-maximum`.
+`VisibilityBox` and `CollisionBox` boundaries is always stay axis aligned even after applying tr_room_staticmesh::Rotation (always have 90 degrees step)
```

Comment:
@XProger If you could add the comment above to an appropriate place, this merge request can be merged.